bears/general/CPDBear.py

```diff
@@ -41,7 +41,7 @@ class CPDBear(GlobalBear):
     def check_prerequisites(cls):  # pragma: no cover
         if which("bash") is None:
             return "bash is not installed."
-        elif which("run.sh") is None:
+        if which("pmd") is None and which("run.sh") is None:
```

Comment:
so what happens when the user doesn't have `pmd`? `which("run.sh")` won't be even evaluated